### PR TITLE
Ignore ABC in specs and serializers

### PR DESCRIPTION
**Pull request by clairewild**
Too noisy

### CODE REVIEW

**Comment by gmkohler** [comment]
💚 🐩  🙉  🐩 💚

File: ruby/.ruby-style.yml [hunk 1]
```diff
@@ -904,7 +904,7 @@ Metrics/AbcSize:
   Exclude:
     - "spec/**/*"
     - "app/serializers/**/*"
-    - "apps/*/app/serializers/**/*"
+    - "apps/*/app/*/serializers/**/*"
```
Comment (binhums) [inline ruby/.ruby-style.yml]: @gmkohler why's this here?

Comment (gmkohler) [inline ruby/.ruby-style.yml]: @binhums https://github.com/zestyzesty/backend/pull/6208/files#r134621943

Comment (binhums) [inline ruby/.ruby-style.yml]: `apps/snacks/app/serializers/snacks/webflow_interface/item_serializer.rb`
`apps/*/app/serializers/**/*_serializer.rb` no?

Am I missing something?

Comment (gmkohler) [inline ruby/.ruby-style.yml]: Are you missing the fact that I'm blind, perhaps?  Will whip up a fix

**Pull request merged**